src/agent/agentIntegration.ts

```diff
@@ -0,0 +1,129 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzExtUserInputWithInputQueue, callWithTelemetryAndErrorHandling, type AgentBenchmarkConfig, type AzureUserInputQueue, type IAzureUserInput, type SimpleCommandConfig, type WizardCommandConfig } from '@microsoft/vscode-azext-utils';
+import { createFunctionApp } from '../commands/createFunctionApp/createFunctionApp';
+
+const createFunctionAppCommandName = "createFunctionApp";
+const createFunctionProjectCommandName = "createFunctionProject";
+const deployToFunctionAppCommandName = "deployToFunctionApp";
+
+export async function getCommands(): Promise<(WizardCommandConfig | SimpleCommandConfig)[]> {
```

Comment:
Note for later: Putting all of the commands `WizardCommandConfigs` in here will definitely get bulky quickly. I'm thinking for organization, we have a commands folder and each file is just a command with all of this information.

Reply:
Sure. We can keep an eye on it.